.github/workflows/_benchmarking_preparation.yml

```diff
@@ -0,0 +1,126 @@
+name: Prepare benchmarking databases by restoring dumps
+
+on:
+  workflow_call:
+    # no inputs needed
+    
+defaults:
+  run:
+    shell: bash -euxo pipefail {0}
+
+jobs:
+  setup-databases:
+    strategy:
+      fail-fast: false
+      matrix:
+        platform: [ aws-rds-postgres, aws-aurora-serverless-v2-postgres ] 
+        database: [ clickbench, tpch, userexample ]
```

Comment:
The disadvantage of having a matrix here is that all dependent jobs must wait until all projects are restored.
If we need to restore only the `tpch` project on `aws-rds-postgres`, all 6 benchmarks will be blocked by the workflow. And a failure in one job will stop all 6 from being run.

I'd probably pass the platform and connstr in inputs instead

Reply:
The matrix can be used because the restore will only be done one time. All future runs will not have to wait until we decide to upgrade the AWS instances to a higher postgres version.
The instances are NOT ephemeral - the restore is only a one time step after new AWS instances are deployed